e2e/cypress/integration/notifications/browser_tab_notification_spec.js

```diff
@@ -0,0 +1,136 @@
+                cy.apiAddUserToTeam(team2.id, user2.id);
+            });
+        });
+    });
+
+    it('MM-T560_1 Browser tab and team sidebar unreads and mentions - Mention in different team', () => {
+        let originalFavicon = '';
+
+        // # User 1 view team A
+        cy.apiLogin(user1);
+        cy.visit(testTeam1TownSquareUrl);
+
+        // # Remove all unreads before test
+        cy.get('#publicChannelList').should('be.visible').get('li').eq(1).click();
+
+        // * Check for no unreads or mentions
+        cy.get('.unread-title').should('not.exist');
+
+        // # Save path of current favicon path
+        cy.get('link[rel=icon]').should('have.attr', 'href').then(($path) => {
+            originalFavicon = $path;
+        });
+
+        // # Have another user view team B
+        cy.apiLogin(user2);
+        cy.visit(testTeam2TownSquareUrl);
+
+        // # Post a message with mentioning User 1 from Town Square
+        cy.postMessage(`@${user1.username}`);
+
+        // # Return to User 1
+        cy.visit(testTeam1TownSquareUrl);
+
+        cy.apiLogin(user1);
+        cy.visit(testTeam1TownSquareUrl);
+
+        // * Browser tab should displays (1) * channel - [team name] Mattermost
+        cy.get('title').
+            contains('(1)');
+
+        // * Team sidebar: Small dot left of team B icon in team sidebar
+        cy.get(`#${team2.name}TeamButton`).parent('.unread').should('be.visible').
+            within(() => {
+                // * Team sidebar: a mention badge in top right corner of the badge with number "1"
+                cy.get('.badge').contains('1').
+
+                // # Make mention indicated
+                    click();
+            });
+
+        // * Favicon should stay blue (not turn red) when there are mentions indicated
+        cy.get('link[rel=icon]').should('have.attr', 'href').then(($path) => {
+            expect($path).to.eq(originalFavicon);
+        });
+    });
+
+    it('MM-T560_2 Team sidebar icon - Badge with mention count increments when added to channel', () => {
+        cy.apiLogin(user1);
+
+        // # User 1 view and remain on team A
+        cy.visit(testTeam1TownSquareUrl);
+
+        // # Have another user view team B
+        cy.apiLogin(user2);
+        cy.visit(testTeam2TownSquareUrl);
+
+        // # Create a new channel
+        cy.get('#createPublicChannel').should('be.visible').click();
+        cy.get('#newChannelName').type('new-channel');
+        cy.get('#submitNewChannel').click();
+
+        // # Invites User 1
+        cy.get('#member_popover').should('be.visible').click();
+        cy.contains('Manage Members').click();
+        cy.contains('Add Members').click();
+        cy.contains(`${user1.username}`).click();
+        cy.get('#saveItems').click();
+
+        // * User 1 should be added.
+        cy.getLastPost().
+            should('contain', `@${user1.username}`).
+            contains('added to the channel');
```

Comment:
This is not the right syntax to verify the last Post. I am seeing the following error. Can you please correct this.

![Screenshot 2020-10-23 at 7 30 44 PM](https://user-images.githubusercontent.com/1429138/97013133-4f869680-1566-11eb-9e86-85c8c1ab2e07.png)

Reply:
I changed with two should assertions. Is there a better way to check whether user is added?